shell.nix

```diff
@@ -29,6 +29,7 @@ mkShell {
     nodejs-10_x
     nvme-cli
     pre-commit
+    python2
```

Comment:
Why do we need python2?

Reply:
Eh... because it is used?  I suspect that it has been working for people because it was already there.  I did most of this work on my VM at work, which runs Ubuntu Server, which does not come with python2 by default, and the tests did not work.  I do not now remember what exactly failed, but it worked again when I added this.  On my VM at home, however, I run Lubuntu, which has python2 by default, so when I took it out it all still worked.  I suggest we keep it here as an explicit entry, because as I said it does appear to be a requisite.

Reply:
Taken out.

Reply:
Removed.